aws-cli-2.yaml

```
@@ -59,6 +65,9 @@ pipeline:
 
 test:
   pipeline:
+    - uses: test/ldd-check
```

nit and non-blocker: we can consider using this more and more going forward. 
```suggestion
    - uses: test/tw/ldd-check
```